Pipfile

```diff
@@ -10,13 +10,14 @@ isort = "*"
 mypy = "==0.660"
 pep8-naming = "*"
 py-spy = "*"
-pylint = "*"
+pylint = "<2.3.0"
```

Comment:
This is necessary due to dependency incompatibilits between `mypy` and `pylint`.
See https://github.com/pypa/pipenv/issues/3568 for details